src/metabase/util/malli.cljc

```diff
@@ -114,21 +116,22 @@
      :cljs string?))
 
 ;; Kondo gets confused by :refer [defn] on this, so it's referenced fully qualified.
-(metabase.util.malli/defn with-api-error-message
-  "Update a malli schema to have a :description (used by umd/describe, which is used by api docs),
+#?(:clj
+   (metabase.util.malli/defn with-api-error-message
+     "Update a malli schema to have a :description (used by umd/describe, which is used by api docs),
```

Comment:
This didn't seem to work in Cljs for whatever reason (not really clear why), but disabling it for Cljs like this did the trick. Maybe because this namespace needs to `:require-macros` itself or something in Cljs mode... I didn't dig into it that much